roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -285,5 +293,10 @@ class ResetNeutron(object):
             return False
 
 
+parser = OptionParser()
+parser.add_option("-d", "--device", type="string", dest="device", help="devices to be reset. eg. vxlan,bridge,tap")
```

Comment:
Consider using `argparse` rather than optparse, which is deprecated and will be going away.  That will make this easier to port in the future.

Additionally, check line length and re-indent as appropriate per pep8.  

Try using the `pep8` or the very good `flake8` tool for assistance.